### ISSUE TITLE
Does Cloud.Log/Logf work with Cloud.Parallel?

### ISSUE THREAD

**Issue by dsyme** [mbraceproject/MBrace.Azure#27]
I was running some Cloud.Parallel code that uses Cloud.Log and got this failure along the way - is there a contention problem here on the Azure storage used for logging?

```
System.AggregateException: One or more errors occurred. ---> Microsoft.WindowsAzure.Storage.StorageException: The remote server returned an error: (409) Conflict. ---> System.Net.WebException: The remote server returned an error: (409) Conflict.
   at Microsoft.WindowsAzure.Storage.Table.Protocol.TableOperationHttpResponseParsers.TableOperationPreProcess(TableResult result, TableOperation operation, HttpWebResponse resp, Exception ex)
   at Microsoft.WindowsAzure.Storage.Core.Executor.Executor.EndGetResponse[T](IAsyncResult getResponseResult)
   --- End of inner exception stack trace ---
   at Microsoft.WindowsAzure.Storage.Core.Executor.Executor.EndExecuteAsync[T](IAsyncResult result)
   at Microsoft.WindowsAzure.Storage.Core.Util.AsyncExtensions.<>c__DisplayClass1`1.<CreateCallback>b__0(IAsyncResult ar)
   --- End of inner exception stack trace ---

   at MBrace.Continuation.ExceptionDispatchInfoUtils.Async.RunSync[T](FSharpAsync`1 workflow, FSharpOption`1 cancellationToken) in c:\Users\eirik\Development\nessos\MBrace.Core\src\MBrace.Core\Continuation\ExceptionDispatchInfo.fs:line 139
   at MBrace.Azure.Runtime.Common.ProcessLogger.MBrace-Runtime-ICloudLogger-Log(String entry) in c:\Users\eirik\Development\nessos\MBrace.Azure\src\MBrace.Azure.Runtime.Common\Records\Logging.fs:line 151
   at MBrace.Azure.Runtime.Common.LoggerCombiner.MBrace-Runtime-ICloudLogger-Log(String entry) in c:\Users\eirik\Development\nessos\MBrace.Azure\src\MBrace.Azure.Runtime.Common\Records\Logging.fs:line 36
   at <StartupCode$MBrace-Core>.$RuntimeCombinators.Log@89-1.Invoke(FSharpOption`1 _arg4) in c:\Users\eirik\Development\nessos\MBrace.Core\src\MBrace.Core\Runtime\RuntimeCombinators.fs:line 92
   at <StartupCode$MBrace-Core>.$CloudBuilder.Bind@209-1.Invoke(ExecutionContext ctx, T t) in c:\Users\eirik\Development\nessos\MBrace.Core\src\MBrace.Core\Continuation\CloudBuilder.fs:line 209
   at Cloud.Parallel[T](seq<Cloud<T>> computations)
```

**Comment by dsyme** [comment]
(I haven't got a clean repro yet, just recording the issue here since I saw it)

**Comment by krontogiannis** [comment]
Aha! I pushed this fix for this issue in 96727ab1ec9c5318ede671ffed077ad968bb94ae. So I'm closing this for now.

**Comment by dsyme** [comment]
Super, thanks
